packages/components/src/ConfirmationModal/ConfirmationModal.mdx

```diff
@@ -109,6 +109,38 @@ that presents a confirm modal.
   }}
 </Playground>
 
+## Destructive
+
+If the user is confirming an action that is will destroy, delete, or remove
```

Comment:
Very minor/nitpicky: Can remove "is" from sentence.  So, `If the user is confirming an action that will destroy, delete, or remove`

Reply:
I had that doubt too, checked with Grammarly and it gave me okay :P